<ui version="4.0" >
 <class>QueueManagerWidget</class>
 <widget class="QWidget" name="QueueManagerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Queue Manager</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Downloads</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTreeView" name="m_downloads" />
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="KPushButton" name="m_downloads_move_up" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="m_downloads_move_down" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Uploads</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTreeView" name="m_uploads" />
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="KPushButton" name="m_uploads_move_up" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="m_uploads_move_down" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
